File: frontend/turing-machine-web/src/logic/Tapes/TapesUtilities/WriteOperation.ts
export class WriteOperation {
    Position: number;
    Content: string;
    MachineID: number;
    HeadID: number;

    constructor(position: number, content: string, machineID: number, headID: number)
    {
        this.Position = position;
        this.Content = content;
        this.MachineID = machineID;
        this.HeadID = headID;
    }
}

File: archive/logic/TuringMachineSimulationNew/Tapes/TapesUtilities/WriteOperation.cs
namespace TuringMachineSimulation.Logic.Tapes.TapesUtilities
{
    public class WriteOperation
    {
        public int Position { get; set; }
        public char Content { get; set; }
        public int MachineID { get; set; }
        public int HeadID { get; set; }
        
        public WriteOperation(int position, char content, int machineID, int headID)
        {
            Position = position;
            Content = content;
            MachineID = machineID;
            HeadID = headID;
        }
    }
}